shared/components/NextConferenceEvent/index.js

```diff
@@ -1,16 +1,53 @@
 import React, { PropTypes } from 'react';
 import { ExternalLink } from '../ExternalLink';
-
+import { ticketType } from '../../prop-types/ticket-type';
 const locationURL = 'https://goo.gl/maps/GkqTFrJKaUR2';
+const conferenceTicketURL = 'https://ti.to/';
```

Comment:
I had the same thought but I don't think we will ever change this link ever once we have it.